lib/services/android-project-service.ts

```diff
@@ -104,6 +105,24 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 		return this._platformData;
 	}
 
+	private getDeviceBuildOutputPath(currentPath: string, projectData: IProjectData): string {
+		const currentPlatformData: IDictionary<any> = this.$projectDataService.getNSValue(projectData.projectDir, constants.TNS_ANDROID_RUNTIME_NAME),
+			platformVersion = currentPlatformData && currentPlatformData[constants.VERSION_STRING],
```

Comment:
I'd prefer each `const` be on a separate line here - this way each variable declaration is a separate statement and debugging is easier and overall it seams neater to me